### PR TITLE
Multi

### PR DESCRIPTION
**Pull request by ritvik-32**
## Release Notes

- **New Features**
	- Introduced comprehensive documentation for deploying a multi-node blockchain network in `multinode/README.md`.
	- Added an `inventory.ini` file to define configuration for multi-node deployments.
	- Created structured playbooks for managing validator and full node setups.
	- Implemented load balancers for RPC and API services.
	- Added configuration templates for explorer and faucet applications.

- **Bug Fixes**
	- Removed outdated `README.md` file that contained no longer relevant information.

- **Chores**
	- Added various service unit files for managing the faucet and node services.

### CODE REVIEW

File: multinode/templates/explorer-config.j2 [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "api": [
+        {"address": "http://{{ lb2_ip }}"}
+    ],
+    "rpc": [
+        {"address": "http://{{ lb1_ip }}"}
+    ],
+    "sdk_version": "0.50.3",
+    "coin_type": "118",
+    "min_tx_fee": "800",
+    "addr_prefix": "cosmos",
```
[the remark this answers is not in the document]
Comment (ritvik-32) [reply multinode/templates/explorer-config.j2:12]: it is hardcoded for that version

File: multinode/templates/explorer-config.j2 [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+    "chain_name": "{{ chain_id }}",
+    "api": [
+        {"address": "http://{{ lb2_ip }}"}
+    ],
```
[the remark this answers is not in the document]
Comment (ritvik-32) [reply multinode/templates/explorer-config.j2:5]: It consists of multiple API addresses

File: multinode/templates/explorer-config.j2 [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    "rpc": [
+        {"address": "http://{{ lb1_ip }}"}
+    ],
+    "sdk_version": "0.50.3",
+    "coin_type": "118",
+    "min_tx_fee": "800",
+    "addr_prefix": "cosmos",
+    "logo": "/logos/cosmos.svg",
```
[the remark this answers is not in the document]
Comment (ritvik-32) [reply multinode/templates/explorer-config.j2:13]: that is the fixed path.

File: multinode/templates/explorer-config.j2 [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    "assets": [{
+        "base": "uatom",
+        "symbol": "ATOM",
+        "exponent": "6", 
+        "logo": "/logos/cosmos.svg"
+    }]
```
[the remark this answers is not in the document]
Comment (ritvik-32) [reply multinode/templates/explorer-config.j2:19]: Its meant to be hardcoded.

**Comment by kaustubhkapatral** [comment]
LGTM. Great work @ritvik-32

**Pull request merged**